include/unifex/let_with_stop_source.hpp

```diff
@@ -164,10 +165,11 @@ struct _stop_source_operation<SuccessorFactory, Receiver>::type {
 
     template<class F>
     using callback_type = typename stop_token_type_t<Receiver>::template callback_type<F>;
+    UNIFEX_NO_UNIQUE_ADDRESS remove_cvref_t<SuccessorFactory> func_;
     UNIFEX_NO_UNIQUE_ADDRESS unifex::inplace_stop_source stop_source_;
     UNIFEX_NO_UNIQUE_ADDRESS callback_type<_stop_source_operation_callback> stop_callback_;
     connect_result_t<
-        std::invoke_result_t<SuccessorFactory, unifex::inplace_stop_source&>,
+        std::invoke_result_t<SuccessorFactory&&, unifex::inplace_stop_source&>,
```

Comment:
This can also be a `callable_result_t` since `SuccessorFactor` will never be a pointer to a member on an `inplace_stop_source`.